index.html

```diff
@@ -234,8 +234,95 @@ <h4 class="work-canopy">canopy</h4>
               </div>
               <button class="tech-project-btn" type="submit">See project</button>
             </div>
-          </section>
+          </section> -->
+        </section>
+        <section class="popup-Container">
+          <!-- <div class="pop-up">
+            <div class="popup-info">
+              <h5 class="popup-h5">tonic</h5>
+              <div class="pop-stack">
+                 <div class="pop-stackname">
+                  <p>canopy</p>
+                  <p>back-end</p>
+                  <p>2015</p>
+                 </div>
+                
+                <div class="popup-img">
+                  <img src="./images/work-1.png" alt="" class="popup-image">
+                </div>
+                <div class="popup-description">
+                 <div class="popup-p">
+                <p class="popup-pone">  Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s. Lorem Ipsum is simply dummy text of the printing and typesetting industry.</p>
+                <p>Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it 1960s with the releorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum han printer took a galley of type and scrambled it 1960s with the releawn printer took a galley of type and scrambled it 1960s</p>
+               
+                 </div>
+                 <div >
+                 <div class="popup-techstack">
+                  <p>html</p>
+                  <p>html</p>
+                  <p>html</p>
+                 </div>
+                <div class="popup-btn">
+                  <button type="submit" class="liveview-btn"><span><img src="./images/Ellipse.png" alt=""></span>see live</button>
+                  <button type="submit" class="liveview-btn">see source</button>
+                  <div class="buttons">
+                    <a href="#" class="see-live"
+                      ><img src="./images/Ellipse.png" alt="see live"
+                    /></a>
+    
+                </div>
+                    
+                    </div>
+                </div>
+              </div>
+            </div>
+          </div> -->
```

Comment:
- [ ] Please do not commit old pieces of code as inline comments. They make your project look messy. If you need to review a previous version of your code, you can always use git history. 🗒️